Packs/MicrosoftCloudAppSecurity/ReleaseNotes/1_0_14.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Microsoft Cloud App Security
+- Improving documentation - added example for the **Custom Filter** parameter, and explanation about the the type and permissions assigned to the user/token used for the integration.
+- Upgraded the Docker image to demisto/python3:3.9.1.15759.
```

Comment:
- Updated the Docker image to: *demisto/python3:3.9.1.15759*.

Reply:
done